functions/supplier/file-supplier/src/test/java/org/springframework/cloud/fn/supplier/file/AbstractFileSupplierTests.java

```diff
@@ -57,5 +62,17 @@ public static void afterAll() {
 
 	@SpringBootApplication
 	static class FileSupplierTestApplication {
+
+		@Bean
+		static IntegrationComponentCustomizer<FileInboundChannelAdapterSpec> fileInboundChannelAdapterSpecCustomizer() {
+			return adapterSpec -> adapterSpec.watchEvents(FileReadingMessageSource.WatchEventType.DELETE);
+		}
+
+		@Bean
+		static IntegrationComponentCustomizer<Date> fakeCustomizer() {
+			return d -> {};
```

Comment:
Wdyt about throwing a runtime exception here instead which would prove that this never gets called (as it shouldn't).